pkg/config/application_config.go

```diff
@@ -112,27 +102,9 @@ type AnnounceableAddress struct {
 }
 
 // GetAddresses parses returns the list of AnnounceableAddress containing information
-// gathered from both deprecated Address / NodePort / AnnouncedNodePort and newly
-// created Addresses fields.
+// gathered from Addresses.
 func (a *ApplicationConfiguration) GetAddresses() ([]AnnounceableAddress, error) {
 	addrs := make([]AnnounceableAddress, 0, len(a.P2P.Addresses)+1)
```

Comment:
Then capacity can be reduced by 1 (no need `+1`).

Reply:
Fixed.